.github/workflows/export_database_views.yml

```diff
@@ -0,0 +1,77 @@
+name: Export Moped Views to SQL
+
+on:
+  push:
+    branches-ignore:
+      - "main"
+      - "production"
```

Comment:
For testing in your personal repo, you can remove these 3 lines to make the workflow trigger on any push. This is handy for development too.